plat/marvell/a8k/common/ble/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Do we really need this file? I think that the one that the rest of the firmware has applies here, specially when you are also going to put it in the headers of each file. This file is redundant.

Reply:
Not, it was maybe required for out of the tree sources. I will remove it.